src/AWS/CRT/Auth/SigningConfigAWS.php

```diff
@@ -34,7 +35,9 @@ public function __construct(array $options = []) {
         $sc = $this->acquire(self::$crt->signing_config_aws_new());
         self::$crt->signing_config_aws_set_algorithm($sc, $options->algorithm->asInt());
         self::$crt->signing_config_aws_set_signature_type($sc, $options->signature_type->asInt());
-        self::$crt->signing_config_aws_set_credentials_provider($sc, $options->credentials_provider->asObject());
+        if ($credentials_provider = $options->credentials_provider->asObject()) {
+            self::$crt->signing_config_aws_set_credentials_provider($sc, $credentials_provider->native);
```

Comment:
92 characters so it should be split among a few lines:
`self::$crt->signing_config_aws_set_credentials_provider(\n
      \t$sc,\n
      \t$credentials_provider->native\n
);`

Reply:
We need to pick a linter. This is going to kill me. This is what tools are for :)

Reply:
Fixed it anyway